chainer/testing/training.py

```diff
@@ -6,7 +6,7 @@
 
 
 def get_trainer_with_mock_updater(
-        stop_trigger=(10, 'iteration'), iter_per_epoch=10):
+        stop_trigger=(10, 'iteration'), iter_per_epoch=10, extend=[]):
```

Comment:
I think the argument name should basically be a noun. In this case, `extensions=` is better.